lisa/notifiers/html.py

```diff
@@ -8,7 +8,7 @@
 import pytest
 from _pytest.config import Config
 from _pytest.reports import CollectReport, TestReport
-from dataclasses_json import dataclass_json  # type: ignore
+from dataclasses_json import dataclass_json
 from pytest_html.plugin import HTMLReport  # type: ignore
```

Comment:
Are we keeping the comment in the line 12?

Reply:
Yes, apparently that is a legit ignore, as there are no stub type annotations for that package.